### PR TITLE
build query string through concatenation

### PR DESCRIPTION
**Pull request by ros3bud**
@alexkwolfe, this update more safely builds the query string and addresses your concern in #4.

### CODE REVIEW

File: src/outbound.coffee [hunk 1]
```diff
@@ -98,6 +98,25 @@ response.variables = ->
...
+getAddress = (vars) ->
+  address = ''
+  count = 0
+  for key in addressKeys
+    if vars.lead[key]
+      if count isnt 0
+        address += " "
+      address += "#{vars.lead[key]}"
+    count += 1
+  address
```
Comment (alexkwolfe) [inline src/outbound.coffee]: This produces an extraneous space when `address_1` is falsey (`" Austin TX"`).

How about something like this?

```coffeescript
getAddress = (vars) ->
  lead = vars.lead
  _([ lead.address_1, lead.city, lead.state, lead.postal_code ])
    .map (val) -> val?.trim()
    .compact()
    .join(' ')
```

Comment (alexkwolfe) [inline src/outbound.coffee]: Consider adding some tests for this...

Comment (ros3bud) [inline src/outbound.coffee]: Updated based on feedback.

**Comment by ros3bud** [comment]
Further updated based on your feedback. LGTY?

**Comment by alexkwolfe** [comment]
LGTM

**Pull request merged**